tokio-util/src/future.rs

```diff
@@ -0,0 +1,125 @@
+//! An extension trait for Futures that provides a variety of convenient adapters.
+
+mod with_cancellation_token;
+
+use std::future::Future;
+
+use with_cancellation_token::{WithCancellationTokenFuture, WithCancellationTokenFutureOwned};
```

Comment:
```suggestion
mod with_cancellation_token;
use with_cancellation_token::{WithCancellationTokenFuture, WithCancellationTokenFutureOwned};

use std::future::Future;
```

nit: we prefer make the import statement more closer to the relevant `mod` statement.